### ISSUE TITLE
Minor: Project Views doc unclear what is means with "we recommend using import"

### ISSUE THREAD

**Issue by vorburger** [salesforce/bazel-vscode-java#112]
It's not clear to me as a reader what this on https://github.com/salesforce/bazel-eclipse/blob/main/docs/common/projectviews.md means:

> Instead of using only the `.eclipse/.bazelproject` file we recommend using `import`.
Simply add an `import` statement for any  `.bazelproject` file you want to use.

_Use_ `import` how? What's _import_ - a command? Where would one put an `import` _statement,_ how?

@guw ?

**Comment by guw** [comment]
@vorburger Does the public documentation help?
https://ij.bazel.build/docs/project-views.html#import

**Comment by vorburger** [comment]
@guw oh, I see... I've raised https://github.com/salesforce/bazel-eclipse/pull/496 to make that a tiny bit clearer for the next person who comes along.

Now I understand WHAT it is and HOW once would use `import`, but I still don't grok WHY one would?

I've attempted to apply this in https://github.com/enola-dev/enola/pull/638. Is that right, would would make my project be able to better imported in IJ?

As-is right now, that PR has no `import` - is that the idea here? (If added, won't it just "follow" the `import` - so same again.) So it's hard-coded to load `//.bazelproject` AND `tools/eclipse/.managed-defaults.bazelproject`, even if that's not `import`? If so, perhaps I can raise another PR to clarify this more.
